Content.Shared/ItemRecall/SharedItemRecallSystem.cs

```diff
@@ -83,7 +83,8 @@ private void RecallItem(Entity<RecallMarkerComponent?> ent)
         if (TryComp<EmbeddableProjectileComponent>(ent, out var projectile))
             _proj.EmbedDetach(ent, projectile, actionOwner.Value);
 
-        _popups.PopupPredicted(Loc.GetString("item-recall-item-summon", ("item", ent)), actionOwner.Value, actionOwner.Value);
+        _popups.PopupPredicted(Loc.GetString("item-recall-item-summon", ("item", ent)), Loc.GetString("item-recall-item-summon-witness", ("item", ent), ("name", actionOwner)), actionOwner.Value, actionOwner.Value);
```

Comment:
```suggestion
        _popups.PopupPredicted(Loc.GetString("item-recall-item-summon", ("item", ent)), Loc.GetString("item-recall-item-summon-witness", ("item", ent), ("name", Identity.Entity(actionOwner, EntityManager))), actionOwner.Value, actionOwner.Value);
```
You will have to pass the user's identity instead or the popup will reveal their identity even if it is hidden by a mask.